app/src/main/java/uk/gov/onelogin/login/state/StateGenerator.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.onelogin.login.state
+
+import java.util.UUID
+import javax.inject.Inject
+
+class StateGenerator @Inject constructor() : IStateGenerator {
```

Comment:
How do you feel about the name `UUIDStateGenerator`? I feel like we've got the power to name it to have a little more context. Especially with it being a little class, yeah.

Reply:
In jira ticket